src/FrameRipper.py

```diff
@@ -0,0 +1,111 @@
+import cv2
+import numpy as np
+import os
+import random
+
+# Frame Ripper Script WIP
+# Working on adding subset functionality in order to randomly grab sets of frames instead of all of them
+
+__author__ = 'Steve'
+
+# Function to scan the video files, processes and adds them to Array video_list
+def get_video_files(video_path = 'Videos'):
+    
+    entries = os.listdir(video_path)
+    video_list = []
+    
+    for entry in entries:
+        video_list.append(video_path + "/" + entry)
+    return video_list
+
+# Function that takes each element from the video_list and returns the total frame count of each video file
+def get_total_frames(videos = get_video_files()):
+    
+    total_frames = []
+    
+    for index, file_name in enumerate(videos):
+        cap = cv2.VideoCapture(file_name)
+        frame_count = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
+        total_frames.append(frame_count)
+    return total_frames
+
+# Video Processing
+def process_video():
+    
+    # howdy partner
+    # video shit goes here yo
+    
+    return
+
+# Sub-set function
+def sub_set_finder(cap, frames):  
+    
+    # Gets the total frames from the video
+    for index, element in enumerate(frames):
+        total_frames = element
+                    
+        # Sets the subset of frames to capture
+        sub_set = int(random.random() * total_frames)
```

Comment:
should this be one value or an array of values, either append to a list or use `yield`

Reply:
another fancy way (list comprehension) to do it is something like
```python
subset = [int(random.random() * total_frames) for total_frames in frames]
```